src/generators/AbstractRenderer.ts

```diff
@@ -81,4 +81,15 @@ export abstract class AbstractRenderer<
     }
     return content;
   }
+
+  getConstValue(property: ConstrainedObjectPropertyModel): ConstrainedEnumValueModel | undefined {
```

Comment:
Really not a fan that we have to use support functions like here to get the constant value, but on the other hand, I don't see a way around it, as it's not really possible to have a `contantValue` property in the `MetaModel` class or `ConstrainedObjectPropertyModel` ...